.github/workflows/scheduled.yml

```diff
@@ -203,8 +203,9 @@ jobs:
 
       - name: Build
         env:
-          EXTRA_CMAKE_FLAGS: >
+          EXTRA_CMAKE_FLAGS: |
             -DVELOX_ENABLE_ARROW=ON \
+            -DVELOX_ENABLE_GEO=ON \
```

Comment:
Ah good catch! I removed the `\` in a later commit but must have lost that when rebasing :facepalm: 
While your fix works removing the `\` is the more idiomatic approach imo. Would you mind changing it to that? I think we can keep the change in this PR :)